PowerFGT/Public/cmdb/firewall/address.ps1

```diff
@@ -57,8 +54,53 @@ function Add-FGTFirewallAddress {
         [Parameter(Mandatory = $false)]
         [psobject]$connection = $DefaultFGTConnection
     )
+DynamicParam
```

Comment:
Can you use ParametSetName ? (i think, don't need DynamicParam)

Reply:
I used paramtersets in my first attempt, but it will break the downwards compatibility with youre current functions. Because -type ipmask or -type fqdn is used its not posible to select the right parameterset. If we use -ipmask (of the type switch) and -fqdn (of the type switch) we could go for the parametersets.

Reply:
Ok, i think it will be more easy to use ParameterSet and no really a problem if downwards the compability ;-) (we can try to kept -type with ParameterSet but need to a check...)